templates/named.conf.erb

```diff
@@ -24,5 +24,12 @@ acl "<%= acl_name %>"  {
 };
 <%- end -%>
 
+<%- if scope.lookupvar('::dns::additional_directives').any? -%>
+// additional directives
+<% scope.lookupvar('::dns::additional_directives').each do |directive| -%>
+<%= directive %>
+<%- end -%>
+<%- end -%>
```

Comment:
I realize that other blocks don't do this either, but I wonder if it should include the empty line so it won't add an empty line for users who don't use this directive. Then it would look like this:

```erb
<%- end -%>

<%- end -%>
// Public view read by Server Admin
```